reflex-dom-core/src/Reflex/Dom/Old.hs

```diff
@@ -112,6 +113,7 @@ type MonadWidgetConstraints t m =
   , MonadSample t (Performable m)
   , MonadReflexCreateTrigger t m
   , PostBuild t m
+  , MonadMountStatus t m
```

Comment:
my understanding of `MonadWidget` is that it's basically everything you can do while running live in a page, and corresponds to all the powers of `Widget` (from `Reflex.Dom.Main`). I can remove it, though I'm not sure what value that has until there's a better solution to all the powers.

can remove it if you want, though. it's trivial to add it as a requirement to use sites (and their callers)